tests/test_fastapi.py

```diff
@@ -142,3 +142,45 @@ def _json_serial(obj):
 
     class Foo(JSONSerialResponse):
         json_serial = _json_serial
+
+
+def test_update_array_metadata(app):
+    from fastapi.openapi.utils import get_openapi
+    from tracktolib.api import Endpoint, CamelCaseModel, add_endpoint
+    from tracktolib.tests import assert_equals
+
+    endpoint_a = Endpoint()
+    endpoint_b = Endpoint()
+
+    class Foo(CamelCaseModel):
+        foo: int
+
+    @endpoint_a.get(model=list[Foo])
```

Comment:
add a different return code and verify it is not overridden

Reply:
also add example and verify it is not overridden: https://fastapi.tiangolo.com/tutorial/schema-extra-example/?h=exampl#extra-json-schema-data-in-pydantic-models